.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
+about: Propose a talk or workshop for upcoming meetup/event
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- You can submit any talks which are related to Python Programming -->
+<!-- language, new libraries learned or created something awesome projects -->
+<!-- or something new in Data Science, Machine Learning, Automation, Cloud, -->
+<!-- Containers, Hardware. -->
+
+**Title of the talk**
+<!-- Please include a short title -->
+
+**Description**
+<!-- Please include brief abstract about your talk. -->
+
+**Prerequisites**
+<!-- If you have any specific requirement from audience for the talk i.e. software, tools set up on machine etc, please include it here -->
```

Comment:
@bhavin192 I think these commented instruction should be visible... maybe like `replace this line` or just in `italic`. what you think?

Reply:
No, these are meant to be kind of instructions for the person who us creating the issue. Those should not be part of the actual issue that's why those are comments.

Reply:
Those will be visible to the person who is filing the issue, so it serves the purpose I would say

Reply:
if those will visible to issue creator then it's ok... I'm just suspect that instruction will not visible.

Reply:
I think Prerequisites should be given in check list format?